extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryProcessor.java

```diff
@@ -94,6 +98,15 @@ public boolean test(AnnotationInstance annotationInstance) {
     private static final DotName ADD_SPAN_ATTRIBUTES_INTERCEPTOR = DotName
             .createSimple(AddingSpanAttributesInterceptor.class.getName());
 
+    @BuildStep(onlyIfNot = MetricProcessor.MetricEnabled.class)
+    void disableMetricLogExporter(BuildProducer<LogCategoryBuildItem> log) {
+        if (isClassPresentAtRuntime(
+                "io.opentelemetry.exporter.logging.LoggingMetricExporter")) {
+            log.produce(new LogCategoryBuildItem(
+                    "io.opentelemetry.exporter.logging.LoggingMetricExporter", Level.OFF));
+        }
```

Comment:
There's no better way to disable it than turning off logging? I would have expected we could avoid registering it?

Reply:
Unfortunately this is not produced by instrumentation but by an exporter:
https://github.com/open-telemetry/opentelemetry-java/blob/3ef4be2d845ba3f960d8b06c6fc85c7dcc6e6c78/exporters/logging/src/main/java/io/opentelemetry/exporter/logging/LoggingSpanExporter.java#L65

Reply:
@rsvoboda, `quarkus.otel.metrics.enabled=false` =  `@BuildStep(onlyIfNot = MetricProcessor.MetricEnabled.class)`
Also, `io.opentelemetry.exporter.logging.LoggingMetricExporter` will only be present at runtime if `quarkus.otel.metrics.exporter=logging`. 
The checks are being made.

Reply:
Will see what I can do

Reply:
I'm out of ideas on this. 
This is loaded from the OTel exporter SPI and we have no way to block the instantiation.
Created this: https://github.com/open-telemetry/opentelemetry-java/pull/7312
I can use reflection to register a config and disable the instrumentation but given this is for tests and debug only, I think the current solution is good enough.

Reply:
I found an easier way around.